components/Quote/Quote.tsx

```diff
@@ -42,8 +43,9 @@ export const Quote = ({
   children,
   className,
   ...props
-}: QuoteProps) => (
-  <AnimateInView animation={animation} delay={delay}>
+}: QuoteProps) => {
+  const content = typeof body === 'string' ? body : <RichText wysiwyg={body} />;
```

Comment:
This one took a while to debug. 

I'm not sure how, but the body in the `stories/education-for-a-life-of-purpose` is coming through looking like 

```
{ type: 'doc', content: [ { type: 'paragraph', content: [Array] } ] }
```

Perhaps a field change?